### PR TITLE
fixed @mkhoshle's name in docs

### PR DESCRIPTION
**Pull request by orbeckst**
Changes made in this Pull Request:
 - fixed @mkhoshle's name in docs `conf.py`

### CODE REVIEW

**Comment by orbeckst** [comment]
@mkhoshle please check that this time I got your name right... all you have to do is review the PR and check "approve". I'll do the merging. Thanks.

File: docs/conf.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 # General information about the project.
 project = u'PMDA'
-author = u'Max Linke, Mahzad Khoslessan, Richard J Gowers, Oliver Beckstein'
+author = u'Max Linke, Mahzad Khoshlessan, Richard J Gowers, Oliver Beckstein'
```
Comment (orbeckst) [inline docs/conf.py]: @mkhoshle : the change fixed your name to read "Khoshlessan" (look at the "Files changed" to see the diff).

Is this ok now?

**Comment by kain88-de** [comment]
ping @mkhoshle

**Comment by mkhoshle** [comment]
@orbeckst Look fine to me!

**Pull request merged**